ModelDescriber/ModelDescriberInterface.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+/*
+ * This file is part of the ApiDocBundle package.
+ *
+ * (c) EXSyst
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace EXSyst\Bundle\ApiDocBundle\ModelDescriber;
+
+use EXSyst\Component\Swagger\Schema;
+
+interface ModelDescriberInterface
```

Comment:
I wonder whether this should contain a `supports` method instead of calling all describers to call only the first describer supporting a model.